pangea/services/audit.py

```diff
@@ -599,27 +596,30 @@ def create_sign_envelope(self, event: dict) -> dict:
         return sign_envelope       
 
     def get_buffer_data(self):
-        if not self.cache_buffer:
+        if not self.buffer_data:
+            self.root_id_filename = get_root_filename()
+
             if os.path.exists(self.root_id_filename):
                 try:
                     with open(self.root_id_filename, "r") as file:
-                        file_data = file.read()
-                    self.buffer_data = json.loads(file_data)
-                    self.cache_buffer = True 
+                        self.buffer_data = file.read()
                 except Exception:
                     raise Exception("Error: Failed loading data file from local disk.") 
 
         return self.buffer_data
 
-    def set_buffer_data(self, last_root_enc: str, pending_roots: List[str]):
+    def set_buffer_data(self, last_root_enc: str, pending_roots: t.List[str]):
         buffer_dict = dict()
         buffer_dict["last_root"] = last_root_enc
         buffer_dict["pending_roots"] = pending_roots
 
+        if not self.buffer_data:
+            self.root_id_filename = get_root_filename()
```

Comment:
```suggestion
        root_id_filename = get_root_filename()
```

(no need to be an object member, no need to calculate it when buffer_data is None)